### DISCUSSION TITLE
Default Browser selection for mobile client

### DISCUSSION THREAD

**Discussion by MrBlackBrain** [dailydotdev/daily#717] [Feature Requests]
On the mobile app every article opens on integrated browser. However it would be better if I could open the articles on chrome and cache the site so that I can read them when I am offline.

(alternatively ability to save post to read them later, offline, would be sufficent.)

**Comment by idoshamun** [comment]
Hi @MrBlackBrain,

Our mobile app is actually a PWA bundled as an app, and as far as I'm aware, it's not possible to change this setting with a user setting.
If you are aware of something else, please let me know :)
